api/components/examples/mutual-funds/on_status/on_status-lumpsum-psuccess-fsuccess.json

```diff
@@ -228,7 +208,31 @@
                         }
                     ]
                 }
-            ]
+            ],
+            "quote": {
+                "id": "q1",
+                "price": { "currency": "INR", "value": "3000" },
+                "breakup": [
+                    {
+                        "item": {
+                            "id": "12391",
+                            "fulfillment_ids": ["ff_008"],
+                            "quantity": {
+                                "allocated": {
+                                    "measure": {
+                                        "value": "150.0000",
+                                        "unit": "MF_UNITS"
+                                    }
+                                }        
+                            },
+                            "price": { "currency": "INR", "value": "20.0000", "listed_value": "20.0000" },
+                            "time": { "label": "NAV Date", "timestamp": "2024-07-23T00:00:00+05:30" }
+                        },
+                        "title": "Investment",
+                        "price": { "currency": "INR", "value": "3000" }
+                    }
+                ]
+            }
```

Comment:
@92shreyansh We have discussed to update the item in order with the details about units allotted and the nav and other details. But as I was making the changes, it felt easy if these are added to the item in the breakup of quote object as we have the ability to add full item object. So kept it here, let me know if this is fine or is there anything I am missing.

Reply:
One of the key reason why I would avoid this is that the item inside the quote is a refrencing element for items of deatailng associated to that breakup.

But if you look at the overall transaction poin of view these are my final SKU purchased in this transaction/order the only difference is the entire defination of the SKU is completed post confirmation. So would be ideal if we update the base item rather than its reference point in breakup.  unless you feel there is some limitation in updating in the base item.

Reply:
Got the point, that makes sense. Updated the examples.

One observation in updating the base item is regarding the `time` attribute. In SIP orders, we are using `time` attribute to capture the sip schedule in `frequency` ([select example](https://github.com/ONDC-Official/ONDC-FIS-Specifications/blob/fis14-discussion/api/components/examples/mutual-funds/select/select-sip.json#L37)). This should not be a problem because in SIP orders, we are not populating the nav details. But do you think there is a dual usage for the time attribute?

Reply:
Are we not using the fufillment stops time to determine the frequency as this will avoid dual usage of the same field and I also found this is a dicripancy as the value sent in select is not flowing back in on_select which we should avoid as far as possible.

Reply:
We were not using fulfillment stops. Now since we have the dual usage, I’ve moved the frequency capture into stops. Updating the PR.

And, the value sent in select is flowing back in on_select and all the remaining calls. I haven’t found any example where that is missing. Which example are you referring to?